setup.py

```diff
@@ -13,10 +13,7 @@
 
 # Installed by pip install osmosis-driver-interface
 # or pip install -e .
-install_requirements = [
-    # Install squid-py and all its dependencies first
-    'osmosis-on-premise-driver==0.0.4',
-]
+install_requirements = []
```

Comment:
Surely there must be _some_ Python packages that are required to run osmosis-driver-interface (and aren't part of the Python Standard Library)? Looking at `log.py`, it looks like it needs `yaml` (i.e. PyYAML) and `coloredlogs`, for example.

Reply:
Yes, I forgot those. We should add it for the next release.

Reply:
Resolved in https://github.com/oceanprotocol/osmosis-driver-interface/pull/23